tests/std/tests/VSO_0157762_feature_test_macros/test.cpp

```diff
@@ -5,465 +5,431 @@
 
 int main() {} // COMPILE-ONLY
 
-// COMPILER FEATURE-TEST MACROS
+// ATTRIBUTE FEATURE-TEST MACROS.
 
-#ifdef _MSVC_LANG
-#if _MSVC_LANG > 201703L
-#define CXX17_MODE 1
-#define CXX20_MODE 1
-#elif _MSVC_LANG > 201402L
-#define CXX17_MODE 1
-#define CXX20_MODE 0
+#ifdef __has_cpp_attribute
+// Good.

```

Comment:
Ditto (pre-existing) "this isn't a sentence or a floating-point literal so it shouldn't have a period at the end"
```suggestion
// Good
```